### ISSUE TITLE
Fix pep8 warnings under login/

### ISSUE THREAD

**Issue by tkw1536** [OpenJUB/dreamjub#5]
Running check_pep8.py gives: 

```
./login/admin.py:1:1: F401 'django.contrib.admin' imported but unused
./login/backend.py:5:1: E302 expected 2 blank lines, found 1
./login/backend.py:14:1: W293 blank line contains whitespace
./login/backend.py:19:1: W293 blank line contains whitespace
./login/models.py:1:1: F401 'django.db.models' imported but unused
./login/tests.py:1:1: F401 'django.test.TestCase' imported but unused
```

**Comment by thunderboltsid** [comment]
@GiORg1 Another issue your way. It's basic errands but will give you an idea of how our git workflow works.

**Comment by ghost** [comment]
Fixed in #12
